src/indexer/handleEvent.ts

```diff
@@ -565,7 +565,22 @@ async function handleEvent(indexer: Indexer<JsonStorage>, event: Event) {
           `rounds/${event.args.roundAddress}/applications/${applicationId}/votes`
         ).insert(vote);
 
+        const donorPartitionedPath = vote.voter
```

Comment:
I don't understand this, what does it do? I can try it later, reviewing on mobile